Src/Base/AMReX_ParmParse.H

```diff
@@ -552,6 +558,13 @@ public:
     //! Add a key 'name'with value 'ref' to the end of the PP table.
     void add (const char* name,
               const std::string&  val);
+
+    //! keyword for files to load
+    static std::string FileKeyword;
+
+    //! Add keys and values from a file to the end of the PP table.
+    static void addfile (std::string const filename, ParmParse::Table& tab);
```

Comment:
@WeiqunZhang oh, how can I pass a table object here? Our `table()` method is only const, so this does not work:
```
amrex::ParmParse pp;
pp.addfile(filename, pp.table());
```

```
error: binding reference of type ‘amrex::ParmParse::Table&’ {aka ‘std::__cxx11::list<amrex::ParmParse::PP_entry>&’} to ‘const Table’ {aka ‘const std::__cxx11::list<amrex::ParmParse::PP_entry>’} discards qualifiers
   44 |                 pp.addfile(filename, pp.table());
      |                                      ~~~~~~~~^~
```

Reply:
oh, `g_table` internally...